instruments/instruments/util_fns.py

```diff
@@ -54,6 +54,41 @@ def assume_units(value, units):
         value = pq.Quantity(value, units)
     return value
 
+
+def convert_temperature(temperature, base):
+    """
+    convert the temperature to the specified base
+    :param temperature: a quantity with units of Kelvin, Celsius, or Fahrenheit
+    :type temperature: `quantities.Quantity`
+    :param base: a temperature unit to convert to
+    :type base: `unitquantity.UnitTemperature`
+    :return: the converted temperature
+    :rtype: `quantities.Quantity`
+    """
+    # quantities reports equivalence between degC and degK, so a string comparison is needed
```

Comment:
I just had to take a look for myself to see if you were right....and yep, pyquantities treats degC as the same as degK/kelvin. Rescaling between the two doesn't change the value...

This should most definitely be brought up with the pyquantities folk